src/GraphQL.Tests/Subscription/ObservableExtensionsTests.cs

```diff
@@ -272,6 +272,20 @@ public void NullArgumentsThrow()
         });
     }
 
+    [Fact]
+    public async Task ExceptionsDuringSubscribeProduceOnError()
+    {
+        var errorObservable = new ErrorObservable();
+        var observable = errorObservable.SelectCatchAsync<string, string>((s, _) => new(s), (ex, _) => new(new ExecutionError(ex.Message)));
+        using var subscription = observable.Subscribe(Observer);
+        await Observer.WaitForAsync("Error 'ExecutionError'. ").ConfigureAwait(false);
```

Comment:
Maybe modify `SampleObserver.OnError` to track text for exception? So this test will check for "Error 'ExecutionError'. This is a test" - that would be more readable.